File: Service/Implement/DanhMucThoiGianLayMauService.cs
namespace Service.Implement
{
    public class DanhMucThoiGianLayMauService : BaseService<DanhMucThoiGianLayMau, IDanhMucThoiGianLayMauRepository>
    , IDanhMucThoiGianLayMauService
    {
    private readonly IDanhMucThoiGianLayMauRepository _DanhMucThoiGianLayMauRepository;
    public DanhMucThoiGianLayMauService(IDanhMucThoiGianLayMauRepository DanhMucThoiGianLayMauRepository) : base(DanhMucThoiGianLayMauRepository)
    {
    _DanhMucThoiGianLayMauRepository = DanhMucThoiGianLayMauRepository;
    }
    }
    }

File: CMS/src/app/shared/DanhMucThoiGianLayMau.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucThoiGianLayMau } from 'src/app/shared/DanhMucThoiGianLayMau.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucThoiGianLayMauService extends BaseService {
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucThoiGianLayMau";
    }
}

